index.d.ts

```diff
@@ -0,0 +1,17 @@
+import { ConstructorOptions } from "logdna"
+import Transport from "winston-transport";
+
+declare class LogDNATransport extends Transport {
+  constructor(options: LogDNATransport.TransportOptions);
+}
+
+declare namespace LogDNATransport {
+  interface TransportOptions extends Transport.TransportStreamOptions, ConstructorOptions {
+    /** The LogDNA API key. */
+    key: string;
+    /** The name of this transport. */
+    name?: string;
+  }
```

Comment:
@Sytten, can you add `level?: string;` and `index_meta?: boolean;` as well?

Reply:
Since it extends the `TransportStreamOptions`, it will pick those up automatically.